src/js/widget.tip.js

```diff
@@ -203,11 +203,6 @@
         _setOption: function(key, value) {
             var widget = this;
             this._super(key, value);
-            if (key == 'content') {
-                widget._contentUpdate(
-                    $.usosUtils._tooltipster_html(widget.options.content, widget.options._autoWidth)
-                );
-            }
```

Comment:
W opisie masz "fix missing _contentUpdate methods". Czy to znaczy, że metoda _contentUpdate nie istnieje, i dlatego usunąłeś ten fragment?